packages/core/src/indexing/sqlite/__tests__/insertion-order.test.ts

```diff
@@ -103,7 +104,7 @@ describe('backward pagination', () => {
         last: PAGE_SIZE,
         before: beforeCursor,
       })
-      beforeCursor = result.pageInfo.startCursor
+      beforeCursor = result.pageInfo.endCursor
```

Comment:
it's really tough to reason about this stuff, but I'm pretty sure we want to always use `endCursor`, regardless of the iteration direction.  If you look at how the startCursor and endCursor are set [here](https://github.com/ceramicnetwork/js-ceramic/blob/7db84e01f47bb54f811db346bcdf672f29ae7bfc/packages/core/src/indexing/sqlite/insertion-order.ts#L91) and [here](https://github.com/ceramicnetwork/js-ceramic/blob/7db84e01f47bb54f811db346bcdf672f29ae7bfc/packages/core/src/indexing/sqlite/insertion-order.ts#L112) you see that the endCursor is always the 'last' entry in the batch, in whatever order you requested.  So in a forward (newest to oldest) query the `endCursor` will be the oldest entry in the batch, and in a reverse (oldest to newest) query the `endCursor` will be the newest entry in the batch.  In both cases, you want to continue iterating in the same order starting after the `endCursor`

Reply:
For context if that might be useful, I think this is the relevant section of the spec: https://relay.dev/graphql/connections.htm#sec-Edge-order

> [...] ordering must be consistent from page to page, and importantly, The ordering of edges should be the same when using first/after as when using last/before, all other arguments being equal. It should not be reversed when using last/before.

Reply:
> It should not be reversed when using last/before.

wait what?  That seems very in contrast to how we are using them.  @PaulLeCam can you take a look at the `basic-indexing.test.ts` file and see if the way iteration works matches your expectations and understanding of the spec?

Reply:
I left some comments there, would be good to get input from @ukstv as well as we discussed this before and we have the current behavior working well for the clients, so I think the current implementation is correct?